src/components/teamBoxes/TeamOneBox.tsx

```diff
@@ -5,50 +5,53 @@ import { get, ref, set, child, onValue, onDisconnect } from 'firebase/database';
+  const { playerId, username } = useSelector((state: RootState) => state.player);
   const playerOnTeamOneOperativesRef = ref(database, `rooms/${roomId}/team-1/operatives/${playerId}`);
   const dispatch = useDispatch();
   const teamTwoRef = ref(database, `rooms/${roomId}/team-2/`);
   const teamOneOperativesRef = ref(database, `rooms/${roomId}/team-1/operatives/`);
   const teamOneSpymasterRef = ref(database, `rooms/${roomId}/team-1/spymaster/`);
-  const { teamOneOperatives, teamOneSpymaster } = useSelector((state) => state.teamOne);
-  const teamOneRemainingCards = useSelector((state) => state.game.team1RemainingCards);
+  const { teamOneOperatives, teamOneSpymaster } = useSelector((state: RootState) => state.teamOne);
+  const teamOneRemainingCards = useSelector((state: RootState) => state.game.team1RemainingCards);
 
   // On click event for a player to be able to join team-1 team as a operative
   const joinTeamOneOp = async () => {
-    //lines 25 - 38 are checking if the current player is already on a team
+    // lines 25 - 38 are checking if the current player is already on a team
     const teamTwo = await get(teamTwoRef);
     const teamTwoOpsAndSpys = teamTwo.val();
     let teamTwoSpymaster;
     let teamTwoOperatives;
-    //Grabbing team twos info
+    // Grabbing team twos info
     if (teamTwoOpsAndSpys && teamTwoOpsAndSpys.spymaster) {
       teamTwoSpymaster = teamTwoOpsAndSpys.spymaster;
     }
-    //Grabbing team twos info
+    // Grabbing team twos info
     if (teamTwoOpsAndSpys && teamTwoOpsAndSpys.operatives) {
       teamTwoOperatives = Object.keys(teamTwoOpsAndSpys.operatives);
     }
-    //If a player is on team 2, they cannot join this team
+    // If a player is on team 2, they cannot join this team
     if (
       (teamTwoSpymaster && teamTwoSpymaster.playerId === playerId) ||
       (teamTwoOperatives && teamTwoOperatives.includes(playerId))
     ) {
+      // tslint:disable-next-line:no-console
       console.log('Cannot join the other team!');
     } else {
-      //Here we want to check if a player is already a spymaster, so that they cannot join both
+      // Here we want to check if a player is already a spymaster, so that they cannot join both
       await get(teamOneSpymasterRef).then((snapshot) => {
-        //If players already exist as team one spymasters:
+        // If players already exist as team one spymasters:
         if (snapshot.exists()) {
-          //'teamOneSpymasters' sets the spymasers id's to an array
-          const teamOneSpymaster = snapshot.val();
-          //Now we can check if the player is a spymaster, if they are, for now we just console log
-          if (teamOneSpymaster.playerId === playerId) {
+          // 'teamOneSpymasters' sets the spymasers id's to an array
+          const teamOneSpymasterGet = snapshot.val();
+          // Now we can check if the player is a spymaster, if they are, for now we just console log
```

Comment:
should we start cleaning up some of these comments? i think they're from pseudocode prior to writing this logic

Reply:
Sounds good, I'll start takin some out!